activerecord/test/cases/persistence_test.rb

```diff
@@ -47,6 +47,7 @@ def test_fills_auto_populated_columns_on_creation
     record_with_defaults = Default.create
     assert_not_nil record_with_defaults.id
     assert_equal "Ruby on Rails", record_with_defaults.ruby_on_rails
+    assert_not_nil record_with_defaults.virtual_stored_number
```

Comment:
I was wondering if we could be a little bit more self-descriptive and call it something like `rand_number_x_by_ten` or maybe even `rand_number_plus_one` with a default function like `rand_number + 1` 
This way our expectation could look like:
```suggestion
    assert_equal record_with_defaults.rand_number + 1,  record_with_defaults.rand_number_plus_one
```

However I agree that at this point this seems excessive and almost like we are trying to test postgresql

My main intention here is to make the same argument reusable in an "update" test if we ever get `UPDATE` to use returning statement, so a hypothetical test for `update` could look like:

```ruby
record_with_defaults.update(rand_number: 100)
assert_equal 101, record_with_defaults.rand_number_plus_one
```

But that's up to you, it can always be refactored when actually needed

Reply:
Sounds good! Since this is merged that refactor can be rolled in with the `UPDATE` + `RETURNING` pr